Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:1.67 as builder
```

Comment:
Have you tried switching to 1.67.1-alpine?

Reply:
Yes, we did try it but we were getting some build error https://gitlab.ebi.ac.uk/ensembl-web/peregrine-eard/-/jobs/1248849

Reply:
Might not be worth the effort as it only saves <1GB

eard-debian latest 5d7446f83b66 21 seconds ago 3.84GB
eard-alpine latest caaf1a478f55 8 minutes ago 3.09GB

Reply:
By splitting the build and the final image you can get the size down to 863Mb 
eard-alpine          latest    1ef21b56f17c   26 seconds ago   863MB

```
FROM rust:1.67.1-alpine as builder

RUN apk add --no-cache musl-dev

RUN rm -rf /app
RUN mkdir /app
COPY . /app

WORKDIR /app/compiler

RUN cargo build --release

FROM rust:1.67.1-alpine as run

RUN mkdir /app

COPY --from=builder /app/compiler/target/release/eard-compiler /app

ENV PATH=$PATH:/app
```

Reply:
If you change the second image form rust:alpine to just alpine you can get the image size down to 14.2Mb 🎉 
`FROM alpine:3.16 as run`

eard-alpine          latest          584a12d975ef   11 seconds ago      14.2MB

Reply:
Wow! ok I will update and check again.